environment.yml

```diff
@@ -6,5 +6,7 @@ dependencies:
   - pyoccad
   - numpy
   - ambiance
-  - pythonocc-core
   - pythermo
+  - pythreejs
+  - pytest
+  - pythonocc-core
```

Comment:
`pythonocc-core` should come with `pyoccad`
`pytest` is a dev dep

Reply:
cleaned